src/data/Data.js

```diff
@@ -735,15 +736,19 @@ export const WorkspaceData = _.flow(
         .Apps.listAppsV2(workspaceId)
         .then((apps) => {
           const foundApp = resolveWdsApp(apps);
-          if (foundApp?.status === 'RUNNING') {
+          if (foundApp?.status === appStatuses.running.status) {
```

Comment:
Personally, I prefer keeping the string values in anticipation of a TypeScript future. But for JS, I can see the "enum" providing more clarity about what `foundApp.status` represents.